styles/themes/charcoal.css

```diff
@@ -146,7 +146,7 @@
   font-style: normal;
   font-weight: 700;
   font-display: swap;
-  src: url(https://fonts.gstatic.com/s/ebgaramond/v19/~ChoKC0VCIEdhcmFtb25kOgsI9NCduwcVAAAvRCAA.ttf) format('truetype');
+  src: url(https://fonts.gstatic.com/s/ebgaramond/v24/SlGDmQSNjdsmc35JDF1K5E55YMjF_7DPuGi-DPNUAw.ttf) format('truetype');
```

Comment:
Off topic of the PR, not sure we're GDPR compliant, but there is that recent news about leaking ip addresses to google violating data policies
https://termageddon.com/google-fonts-violates-gdpr/

Reply:
Yeah, saw that. pgdp.net is not GDPR compliant with cookies either.

Reply:
I remember this issue coming up once before, but I don't remember where or when. I thought that the answer was that for \<reasons\>, it didn't apply to us, but I can't quote those reasons, and don't know whether the assessment might change over time.